src/server-defaults.jl

```diff
@@ -45,7 +45,11 @@ function jupyterlab_proxy_url(port)
         hostname = config[1]["hostname"]
         # TODO, this seems very fragile
         if hostname == "0.0.0.0"
-            return string("http://", IJulia().profile["ip"], ":", config[1]["port"], "/proxy/", port)
+            hostname = get(ENV, "REMOTE_HOST", "none")
```

Comment:
Could we rename this to something more explicit, like `BONITO_JUPYTER_REMOTE_HOST`?

Reply:
Come to think of it, maybe `BONITO_JUPYTER_REMOTE_HOST` should just always be used if it's set instead of only if the hostname is `0.0.0.0` or `localhost`?

Reply:
I broke it up into two if statements, which I think makes the code a bit easier to understand.